content/en/docs/tasks/traffic-management/egress/egress-kubernetes-services/index.md

```diff
@@ -0,0 +1,268 @@
+weight: 60
+---
+
+Kubernetes [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
+services and Kubernetes services with
+[Endpoints](https://kubernetes.io/docs/concepts/services-networking/service/#services-without-selectors)
+let you create a local DNS _alias_ to an external service.
+This DNS alias has the same form as the DNS entries for local services, namely
+`<service name>.<namespace name>.svc.cluster.local`. DNS aliases provide _location transparency_ for your workloads:
+the workloads can call local and external services in the same way. If at some point in time you decide to deploy the
+external service inside your cluster, you can just update its Kubernetes service to reference the local version. The workloads will continue to operate without any change.
+
+This task shows that these Kubernetes mechanisms for accessing external services continue to work with Istio.
+The only configuration step you must perform is to use a TLS mode other than Istio's
+[mutual TLS](/docs/concepts/security/#mutual-tls-authentication). The external services are not part of an Istio service
+mesh so they cannot perform the mutual TLS of Istio. You must set the TLS mode according to the TLS requirements of the
+external service and according to the way your workload accesses the external service. If your workload issues plain
+HTTP requests and the external service requires TLS, you may want to perform TLS origination by Istio. If your workload
+already uses TLS, the traffic is already encrypted and you can just disable Istio's mutual TLS.
+
+While the examples in this task use HTTP protocols,
+Kubernetes Services for egress traffic work with other protocols as well.
+
+{{< boilerplate before-you-begin-egress >}}
+
+*  Create a namespace for a source pod without Istio control:
+
+    {{< text bash >}}
+    $ kubectl create namespace without-istio
+    {{< /text >}}
+
+*  Start the [sleep]({{< github_tree >}}/samples/sleep) sample in the `without-istio` namespace.
+
+    {{< text bash >}}
+    $ kubectl apply -f @samples/sleep/sleep.yaml@ -n without-istio
+    {{< /text >}}
+
+*   To send requests, create the `SOURCE_POD_WITHOUT_ISTIO` environment variable to store the name of the source
+    pod:
+
+    {{< text bash >}}
+    $ export SOURCE_POD_WITHOUT_ISTIO=$(kubectl get pod -n without-istio -l app=sleep -o jsonpath={.items..metadata.name})
+    {{< /text >}}
+
+*   Verify that the Istio sidecar was not injected, that is the pod has one container:
+
+    {{< text bash >}}
+    $ kubectl get pod $SOURCE_POD_WITHOUT_ISTIO -n without-istio
+    NAME                     READY   STATUS    RESTARTS   AGE
+    sleep-66c8d79ff5-8tqrl   1/1     Running   0          32s
+    {{< /text >}}
+
+## Kubernetes ExternalName service to access an external service
+
+1.  Create a Kubernetes
+    [ExternalName](https://kubernetes.io/docs/concepts/services-networking/service/#externalname) service
+    for `httpbin.org`:
```

Comment:
in the default namespace.

Reply:
Done in https://github.com/istio/istio.io/pull/6206/commits/91b894902204accf9f53602a0531628835e4f21a.